lib/white_bread/formatter/failed_step.ex

```diff
@@ -4,25 +4,28 @@ end
 
 defimpl WhiteBread.Formatter.FailedStep, for: Atom do
   alias WhiteBread.CodeGenerator
+  alias WhiteBread.Outputers.Style
 
   def text(:missing_step, step, _error) do
     %{text: step_text} = step
     code_to_implement = CodeGenerator.Step.regex_code_for_step(step)
-    "undefined step: #{step_text} implement with\n\n" <> code_to_implement
+    Style.info "undefined step: #{step_text}" <>
+    " implement with\n\n" <> code_to_implement
```

Comment:
A bit of a nit-pic  but could you move the ```<>``` to the start of the wrapped line. I prefer this style as it makes it clear that the line is a continuation of the previous line.